ble/UUID.h

```diff
@@ -42,7 +97,7 @@ class UUID {
      *         different service or characteristics on the BLE device.
      *
      * @param longUUID
-     *          The 128-bit (16-byte) UUID value, MSB first (big-endian).
+     *          The 128-bit (16-byte) UUID value, LSB first (little-endian).
```

Comment:
We could. But I think the question is what is user-friendly? Ideally, we would like an API that makes it difficult to use incorrectly. I would find it cumbersome if I had to reverse the endian every time I jumped between software layers. 

Have you had a chance to look at what other BLE stacks are using?